testsuite/domain/src/test/java/org/jboss/as/test/integration/domain/suites/IgnoredResourcesTestCase.java

```diff
@@ -75,6 +83,31 @@ public static void setupDomain() throws Exception {
         testSupport = DomainTestSuite.createSupport(IgnoredResourcesTestCase.class.getSimpleName());
         domainMasterLifecycleUtil = testSupport.getDomainMasterLifecycleUtil();
         domainSlaveLifecycleUtil = testSupport.getDomainSlaveLifecycleUtil();
+
+        final ModelNode slaveModel = DomainTestUtils.executeForResult(Util.getReadAttributeOperation(PathAddress.pathAddress(HOST, "slave"), DOMAIN_CONTROLLER),
+                domainMasterLifecycleUtil.getDomainClient()).get(REMOTE);
+
+        slaveModel.get(IGNORE_UNUSED_CONFIG).set(false);
+        DomainTestUtils.executeForResult(Util.createEmptyOperation("remove-remote-domain-controller", PathAddress.pathAddress(HOST, "slave")), domainSlaveLifecycleUtil.getDomainClient());
+        ModelNode writeRemoteDc = Util.createEmptyOperation("write-remote-domain-controller", PathAddress.pathAddress(HOST, "slave"));
+        for (String key : slaveModel.keys()) {
+            writeRemoteDc.get(key).set(slaveModel.get(key));
+        }
+        DomainTestUtils.executeForResult(writeRemoteDc, domainSlaveLifecycleUtil.getDomainClient());
+
+        ModelNode restartSlave = Util.createEmptyOperation("reload", PathAddress.pathAddress(HOST, "slave"));
+        restartSlave.get(RESTART_SERVERS).set(false);
+        domainSlaveLifecycleUtil.executeAwaitConnectionClosed(restartSlave);
+        domainSlaveLifecycleUtil.connect();
+        domainSlaveLifecycleUtil.awaitHostController(System.currentTimeMillis());
+
+        domainMasterLifecycleUtil = testSupport.getDomainMasterLifecycleUtil();
+        domainSlaveLifecycleUtil = testSupport.getDomainSlaveLifecycleUtil();
+
+        ModelNode slaveIgnore = Util.createEmptyOperation(READ_RESOURCE_OPERATION, PathAddress.pathAddress(HOST, "slave"));
+        ModelNode result = DomainTestUtils.executeForResult(slaveIgnore, domainSlaveLifecycleUtil.getDomainClient());
+        // make sure that ignore-unused-configuration is false
+        Assert.assertFalse(result.get(DOMAIN_CONTROLLER).get(REMOTE).get(IGNORE_UNUSED_CONFIG).asBoolean());
```

Comment:
This needs to be reversed when the test is done, or the rest of the suite will execute with an unexpected config.

After CR2 we should consider separating this test out from DomainTestSuite and just giving it its own config files. Part of the reason for that is a fair bit of the stuff in the config files DomainTestSuite uses may only be there for this test. I'm not sure about that though; it's just a thought.